pages/shared/groupWorkInitial.ejs

```diff
@@ -4,7 +4,7 @@
       <div class="col-sm bg-light py-4 px-4 border">
         <form id="create-form" name="create-form" method="POST">
           <h6>Group name</h6>
-          <input type="text" class="form-control" id="groupNameInput" name="groupName" placeholder="e.g. teamOne" aria-describedby="groupNameHelp">
+          <input type="text" class="form-control" id="groupNameInput" name="groupName" maxlength="10" placeholder="e.g. teamOne" aria-describedby="groupNameHelp">
```

Comment:
We should probably combine this with server-side validation.

Reply:
Thanks for your feedback! The new commit added server-side validation.